__testfixtures__/literal.input.js

```diff
@@ -0,0 +1,5 @@
+import Model, { attr } from 'ember-data/model';
```

Comment:
Here, it seems we have a wrong import. It should be:
```
 import Model from 'ember-data/model';
 import attr from 'ember-data/attr';
```

Reply:
It won't make the test fail, because `attr` is identified in the modules lookup.